### PR TITLE
Upgrade python-telegram-bot version to 20.1

### PR DESCRIPTION
**Pull request by s7904893**

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -60,83 +60,90 @@ class NotifyUserException(Exception):
...
     for elem in json_data:
         meal_notes = elem["notes"]
         markdown_highlight_char = "_"
         for note in meal_notes:
             if "vegetarisch" in note or "vegan" in note:
                 markdown_highlight_char = "*"
 
-        img_url = elem["image"].lstrip("/") # For some reason, image URLs are prefixed with 2 leading slashes, but no protocol, remove them
+        img_url = elem["image"].lstrip(
+            "/")  # For some reason, image URLs are prefixed with 2 leading slashes, but no protocol, remove them
```
Comment (fried-ice) [inline src/main.py]: Unrelated change, please remove

File: src/main.py [hunk 2]
```diff
@@ -241,126 +254,139 @@ def get_post_type(post):
...
+    await context.bot.send_photo(
         chat_id=update.message.chat_id,
         photo=picture_url,
         caption=text
     )
 
 
-def horse(update, context):
-    context.bot.send_photo(
+async def horse(update: Update, context: ContextTypes.DEFAULT_TYPE):
```
Comment (fried-ice) [inline src/main.py:365]: Is there a reason why you indicate types in this method definition but not in others?

**Review by fried-ice — COMMENTED** [review]
You declared all methods as async.
After taking a quick peek at the telegram bot api reference, I think async request handling is not enabled by default. Could you point out the benefit of the requested changes or link the relevant sections of the documentation?

**Comment by s7904893** [comment]
> You declared all methods as async.
> After taking a quick peek at the telegram bot api reference, I think async request handling is not enabled by default. Could you point out the benefit of the requested changes or link the relevant sections of the documentation?

I got an error when not using async and assumed that it has to be done like that for all methods that send anything to the bot (also not all methods are declared as async). I used an example as reference [doc](https://github.com/python-telegram-bot/python-telegram-bot/blob/master/examples/inlinebot.py)

File: src/main.py (hunk 2 above)
Comment (s7904893) [reply src/main.py:365]: No just forgot to add it to all methods

Comment (fried-ice) [reply src/main.py:366]: > > You declared all methods as async.
> > After taking a quick peek at the telegram bot api reference, I think async request handling is not enabled by default. Could you point out the benefit of the requested changes or link the relevant sections of the documentation?
> 
> I got an error when not using async and assumed that it has to be done like that for all methods that send anything to the bot (also not all methods are declared as async). I used an example as reference [doc](https://github.com/python-telegram-bot/python-telegram-bot/blob/master/examples/inlinebot.py)

My bad, I referenced the wrong version of the documentation. Version 20 does in fact require a python coroutine to set up a [`CommandHandler](https://docs.python-telegram-bot.org/en/stable/telegram.ext.commandhandler.html#telegram.ext.CommandHandler.params.callback).

**Comment by fried-ice** [comment]
> 

My bad, I referenced the wrong version of the documentation. Version 20 does in fact require a python coroutine to set up a [CommandHandler](https://docs.python-telegram-bot.org/en/stable/telegram.ext.commandhandler.html#telegram.ext.CommandHandler.params.callback).

File: requirements.txt [hunk 3]
```diff
@@ -1,6 +1,6 @@
-python-telegram-bot==13
+python-telegram-bot
```
Comment (fried-ice) [inline requirements.txt]: Can you please pin the version to a major release again? Looks like the api often introduces breaking changes.

File: src/main.py (hunk 2 above)
Comment (fried-ice) [reply src/main.py:365]: Please settle for one of the two styles
